### DISCUSSION TITLE
Help wanted about cross-compiler

### DISCUSSION THREAD

**Discussion by Saikatsaha1996** [termux/termux-packages#8229] [Q&A] [answered]
Can it possible to cross-compile with clang --target=aarch64-linux-gun?

**Comment by ghost** [comment, marked as answer]
Try following:
* `aarch64-linux-android`
* `aarch64-unknown-linux-android`
* `aarch64-unknown-linux-android24`

You need to have `ndk-multilib` installed.

Termux is Android, but the latter isn't GNU-compatible.

**Reply by Saikatsaha1996** [reply]
Thank you
